pkgs/development/web/nodejs/nodejs.nix

```diff
@@ -140,9 +245,10 @@ let
 
     __darwinAllowLocalNetworking = true; # for tests
 
-    # TODO: what about tests when cross-compiling?
-    # Note that currently stdenv does not run check phase if build ≠ host.
-    doCheck = true;
+    doCheck = canExecute;
+
+    # See https://github.com/nodejs/node/issues/22006
+    enableParallelChecking = false;
```

Comment:
It seems like upstream hasn’t seen any issues here for years and closed this issue as a result. I’m wondering if it really is an upstream issue, but if so, it’d be good to ping the issue with example logs to hopefully get it reopened (and maybe even fixed).